ui/src/app/machines/views/MachineDetails/MachineStorage/FilesystemsTable/FilesystemsTable.tsx

```diff
@@ -1,31 +1,75 @@
 import React, { useState } from "react";
 
 import { Button, MainTable, Tooltip } from "@canonical/react-components";
+import { useDispatch } from "react-redux";
 
+import ActionConfirm from "../ActionConfirm";
 import type { NormalisedFilesystem } from "../types";
 import { formatSize } from "../utils";
 
 import AddSpecialFilesystem from "./AddSpecialFilesystem";
 
+import TableMenu from "app/base/components/TableMenu";
+import { actions as machineActions } from "app/store/machine";
+import type { Machine, MachineStatus } from "app/store/machine/types";
+
+type Expanded = {
+  content: "removeFilesystem";
```

Comment:
`content` can only ever be the string `"removeFilesystem"`? Presumably other types of content will be added in the future?

Reply:
Correct!